packages/stream_chat_flutter/lib/src/message_list_view.dart

```diff
@@ -1218,10 +1226,26 @@ class _MessageListViewState extends State<MessageListView> {
     if (newStreamChannel != streamChannel) {
       streamChannel = newStreamChannel;
       _messageNewListener?.cancel();
-      initialIndex = _initialIndex;
-      initialAlignment = _initialAlignment;
 
-      WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
+      WidgetsBinding.instance!.addPostFrameCallback((timeStamp) async {
```

Comment:
Quick q: Why does this need to be async?

Reply:
it does not 
```suggestion
      WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
```